Makefile

```diff
@@ -118,4 +124,6 @@ docker_push: docker_tag docker_auth ## push to docker hub
 	docker push "openedx/edx-platform:${GITHUB_SHA}"
 	docker push 'openedx/edx-platform:latest-newrelic'
 	docker push "openedx/edx-platform:${GITHUB_SHA}-newrelic"
+	docker push 'openedx/edx-platform:latest-devstack'
+	docker push "openedx/edx-platform:${GITHUB_SHA}-devstack"
```

Comment:
This tagging scheme excludes mention of whether the image is LMS vs. Studio. And I could be misunderstanding, but I think tagging is meant for versioning, not service variants, right?

What about naming the images differently for LMS vs. Studio?
* `openedx/lms`
* `openedx/lms-dev`
* `openedx/lms-newrelic`
* `openedx/studio`
* `openedx/studio-dev`
* `openedx/studio-newrelic`

Reply:
NOTE: This has been ticketed in [ARCHBOM-1440](https://openedx.atlassian.net/browse/ARCHBOM-1440). We'll pick back up on this after we are a bit further along in the prototyping.